### ISSUE TITLE
Gradle publishToMavenLocal fails at generatePomFileForDap4Publication

### ISSUE THREAD

**Issue by bencaradocdavies** [Unidata/thredds#596]
Running:

```
./gradlew publishToMavenLocal
```

on master fails with:

```
[...]
:buildDap4 UP-TO-DATE
:generatePomFileForDap4Publication FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':generatePomFileForDap4Publication'.
> assert pomDependencyNodes*.name()*.localPart.toUnique() == ['dependency']
         |                   |       |         |          |
         []                  []      []        []         false

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED
```

Note that ```gradlew``` just auto-updated itself to 2.14, but this is not the cause as the command succeeds if I roll back to 06c0bef1d4e2ed1bca99a86d6fbee5fce697c27b.

**Comment by cwardgar** [comment]
Fixed in #596. Didn't think about how the POM scope adjustment would affect the rest of our publications. Oops.

@bencaradocdavies Thanks for the report!

**Comment by bencaradocdavies** [comment]
Thanks @cwardgar, confirmed fixed.
